### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by ritazakhodyaeva**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,15 +1,31 @@
...
-<!-- default file list -->
-*Files to look at*:
 
-* [Form1.cs](./CS/WindowsApplication53/Form1.cs) (VB: [Form1.vb](./VB/WindowsApplication53/Form1.vb))
-<!-- default file list end -->
-# How to Change the Field Value Header Background with the CustomDrawFieldValue Event
+# WinForms Pivot Grid - Change the Field Value Header Background
 
-This exampole handles the [PivotGridControl.CustomDrawFieldValue](https://docs.devexpress.com/WindowsForms/DevExpress.XtraPivotGrid.PivotGridControl.CustomDrawFieldValue) event to change the header's backcolor.
+This example handles the [CustomDrawFieldValue](https://docs.devexpress.com/WindowsForms/DevExpress.XtraPivotGrid.PivotGridControl.CustomDrawFieldValue) event to change the header's back color.
```
Comment (dirk-pieterse) [inline Readme.md]: `back color` Is this the same as fill color?

**Review by dirk-pieterse — APPROVED** [review]

File: Readme.md (hunk 1 above)
Comment (ritazakhodyaeva) [reply Readme.md]: yeap

Comment (dirk-pieterse) [reply Readme.md]: Is it not better to use 'fill color' instead since this is more common?

Comment (ritazakhodyaeva) [reply Readme.md]:
```suggestion
This example handles the [CustomDrawFieldValue](https://docs.devexpress.com/WindowsForms/DevExpress.XtraPivotGrid.PivotGridControl.CustomDrawFieldValue) event to fill the header's color. 
```

**Pull request merged**